docker/test/stateful/run.sh

```diff
@@ -36,10 +36,26 @@ echo "ASAN_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer" >> /etc/environ
 echo "UBSAN_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer" >> /etc/environment
 echo "LLVM_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer" >> /etc/environment
 
-service zookeeper start
-sleep 5
-service clickhouse-server start
-sleep 5
+function start()
+{
+    counter=0
+    until clickhouse-client --query "SELECT 1"
+    do
+        if [ "$counter" -gt 120 ]
+        then
+            echo "Cannot start clickhouse-server"
+            cat /var/log/clickhouse-server/stdout.log
+            tail -n1000 /var/log/clickhouse-server/stderr.log
+            tail -n1000 /var/log/clickhouse-server/clickhouse-server.log
+            break
+        fi
+        timeout 120 service clickhouse-server start
```

Comment:
120 retries and 120 seconds timeout. `120 * 120` = 4 hours. Isn't it too much?

Reply:
Yes, it's too much. The same logic is used in stress test.
I don't expect inner timeout will be needed on every loop iteration.

Reply:
ok